pkg/kubernetes/util.go

```diff
@@ -1,20 +1,34 @@
 package kubernetes
 
 import (
+	"k8s.io/client-go/dynamic"
 	"strings"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+type (
+	Kubernetes interface {
+		ResourceContains(namespace, value string, resource schema.GroupVersionResource) (bool, error)
+	}
+	// KubernetesHelper is an implementation of the interface. (Better name? introduced for better testing support)
+	KubernetesHelper struct {
```

Comment:
Is there a reason for this type to be exported? Only the interface should be used outside the package.

Reply:
nope, I forgot about that. Would renaming it to `kubernetesImpl` also make for sense? Any other name?

Reply:
I'm ok with `kubernetesImpl` 👍